plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -517,6 +573,53 @@ SecRule REQUEST_METHOD "@streq PROPFIND" \
         "t:none,\
         ctl:ruleRemoveById=921110"
 
+# Syncing calendars
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/calendars/[^/]+/[^/]+/[^/]+\.ics$" \
+    "id:9508333,\
+    phase:3,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    chain"
+    SecRule RESPONSE_STATUS "@streq 500" \
+        "t:none,\
+        ctl:ruleRemoveById=950100"
+
+# Editing/deleting existing calendar appointments
+# Calendar description can be anything
+SecRule REQUEST_FILENAME "@rx /apps/calendar/v[0-9]/appointment_configs/[0-9]+$" \
+    "id:9508334,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.description,\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT DELETE'"
+
+# Calendar description can be anything
+SecRule REQUEST_FILENAME "@rx /apps/calendar/v[0-9]/appointment_configs$" \
+    "id:9508335,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.description"
+
+# Emptying Calendar trashbin
```

Comment:
Please comment on why you disable 200002.

Reply:
Fixed